core/server/api/canary/stats.js

```diff
@@ -0,0 +1,18 @@
+const statsService = require('../../services/stats');
+
+module.exports = {
+    docName: 'stats',
+    memberCountHistory: {
+        permissions: {
+            docName: 'members',
+            method: 'browse'
+        },
+        async query() {
+            const data = await statsService.members.getCountHistory();
+
+            return {
+                data
```

Comment:
Should we have a `meta` property here? I think we're treating each entry as a resource right? Maybe just something like
```js
"meta": {
    "pagination": {
        "page": 1,
        "limit": "all",
        "pages": 1,
        "total": data.length,
        "next": null,
        "prev": null
    }
}
```

I guess this would be better inside the service 🤔 

Not sure if we need this though

Reply:
Okay, I've added it to the service 👍